de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Purchase_InOut_Customs_Details.sql

```diff
@@ -63,6 +64,17 @@ FROM
 			LEFT OUTER JOIN M_HU_Attribute thuwn ON thu.M_HU_ID = thuwn.M_HU_ID
 				AND thuwn.M_Attribute_ID = ((SELECT M_Attribute_ID FROM M_Attribute WHERE value = 'WeightGross'))
 				AND thuwn.isActive = 'Y'
+			--need LU to calculate the weight
+			LEFT OUTER JOIN M_HU_Assignment lhuas ON iol.M_InOutLine_ID = lhuas.Record_ID AND lhuas.isActive = 'Y' 
+								AND lhuas.AD_Table_ID = ((SELECT Get_Table_ID('M_InOutLine'))) and lhuas.M_HU_Assignment_id != thuas.M_HU_Assignment_id
+			LEFT OUTER JOIN M_HU lhu ON lhuas.M_LU_HU_ID = lhu.M_HU_ID
+			LEFT OUTER JOIN M_HU_PI_Version luv ON lhu.M_HU_PI_Version_ID = luv.M_HU_PI_Version_ID
+			LEFT OUTER JOIN M_HU_PI_Item lhupi ON luv.M_HU_PI_Version_ID = lhupi.M_HU_PI_Version_ID and lhupi.itemtype='PM'
```

Comment:
i think, here (not 100% sure about other places), we *need* to check for `isactive='Y'` because an M_HU_PI_Item coudl jsut be there, but inactive and then it's product's weight would be counted.